app/pages/Settings/Settings.js

```diff
@@ -310,6 +327,26 @@ class Settings extends Component<Props, State> {
       </Flexbox>
     )
   }
+
+  renderWalletMode() {
+    const { walletModeStore } = this.props
+    const [currentWalletMode, otherWalletMode] = walletModeStore.modes
```

Comment:
```suggestion
    const [currentWalletMode, otherWalletMode] = this.props.walletModeStore.modes
```

One liner instead of two?